### DISCUSSION TITLE
📺 Join Live: Wrap Up and Take Action: Join Our Community for the Next Big Thing! 

### DISCUSSION THREAD

**Discussion by kamalshree** [microsoft/hack-together-teams#73] [Announcements]
**Join us live (Thursday, Jun 15) at 4PM GMT/9AM PT for the Wrap UP and Take Action session and meet the Community**

![#6 - June 15 - Wrap Up and Take Action: Join Our Community for the Next Big Thing!](https://github.com/microsoft/hack-together-teams/assets/3199282/e596b5c2-8a46-4b0a-a9e5-15431eb0ac9d)

As the hackathon ends, your journey doesn’t have to. Meet our community members to learn their stories, how they joined the community, learned from others and share their knowledge to give it back. In this session, you’ll meet Microsoft community leads and Microsoft MVPs. Don’t miss this opportunity to learn about our community and what we’ve got to offer! 

Meet our Team, @aycabas @waldekmastykarz @hugoabernier @PopWarner

Join us live and bring any and all questions you might have.

👋 See you there! https://aka.ms/hack-together-teams/session6

**Comment by waldekmastykarz** [comment]
Don't miss this awesome session and meet some of our community leads and community members to hear from them how the community boosted their career!
